nixos/tests/blocky.nix

```diff
@@ -0,0 +1,54 @@
+import ./make-test-python.nix {
+  name = "blocky";
+
+  nodes = {
+    minimalConf = { ... }: {
+      services.blocky = {
+        enable = true;
+
+        settings = {
+          upstream = {
+            default = [ "8.8.8.8" "1.1.1.1" ];
+          };
+          port = 53;
+          httpPort = 4000;
+          logLevel = "info";
+        };
+      };
+    };
+
+    biggerConf = { pkgs, ... }: {
+      environment.systemPackages = [ pkgs.dnsutils ];
+      services.blocky = {
+        enable = true;
+
+        settings = {
+          customDNS = {
+            mapping = {
+              "printer.lan" = "192.168.178.3,2001:0db8:85a3:08d3:1319:8a2e:0370:7344";
```

Comment:
is this a hard coded address? 👀

Reply:
It is based on the [this example](https://github.com/0xERR0R/blocky/blob/64c198529136b07dfa9146c4a545e9481f2e06e7/docs/config.yml#L18-L23).

Reply:
Yeah, just to test that arbitrary custom DNS entries work.